openquake/commonlib/util.py

```diff
@@ -126,3 +127,43 @@ def get_assets(dstore):
     asset_data = [(a['asset_ref'], taxo[a['taxonomy']], a['lon'], a['lat'])
                   for a in assetcol]
     return numpy.array(asset_data, asset_dt)
+
+
+def get_ses_idx(etag):
+    """
+    >>> get_ses_idx("col=00~ses=0007~src=1-3~rup=018-01")
+    7
+    """
+    return int(etag.split('~')[1][4:])
```

Comment:
Perhaps splitting by `=` would make it more robust, in case `ses` is renamed or the number becomes something different than 4 digits. But if it's going to be always like this, this is ok.